src/taglistdelegate.cpp

```diff
@@ -13,9 +13,9 @@ TagListDelegate::TagListDelegate(QObject *parent) :
     m_displayFont(QStringLiteral("Roboto")),
     #endif
     #ifdef __APPLE__
-    m_titleFont(m_displayFont, 13, 65),
+	m_titleFont(m_displayFont, 13, QFont::DemiBold),
     #else
-    m_titleFont(m_displayFont, 10, 60),
+	m_titleFont(m_displayFont, 10, QFont::DemiBold),
```

Comment:
This change makes the `#ifdef` pointless. Someone (probably @nuttyartist) would need to judge whether we still need a slightly more bold title font on macOS.

(this change repeats itself many times, which probably calls for a shared function)

Reply:
The font size is still different, which is why the `#ifdef` is still there. I fully agree with the shared function idea, though. @nuttyartist What is your opinion on this?

Reply:
It looks like many classes initialize the same font options in their constructor. Would it be a better idea to let them inherit from some kind of "Widget with default font" class, so every class that inherits from this gets the same font options (instead of creating a function).

Particularly the `*delegatedEditor`s use exactly the same code, but there are many more classes that use very similar code for fonts.

Reply:
@zjeffer In general, inheritance is not a good option for solving such a problem because we can only inherit from a single class. So you can't inherit from both the "widget with default font" and a "button widget", for example.

Another problem with inheritance is that it would still mean every widget carries its own `QString m_displayFont` and `QFont m_titleFont` members, which is just a waste of computation every time such a widget is instantiated and a waste of memory. The `m_displayFont` isn't even used at all anymore after the `QFont` has been created.

In my opinion it's better to create for example a singleton that will initialize and provide access to the fonts to be used in the application:

```c++
static QString displayFont()
{
#ifdef __APPLE__
    return QFont(QStringLiteral("SF Pro Text")).exactMatch() ? QStringLiteral("SF Pro Text") : QStringLiteral("Roboto");
#elif _WIN32
    return QFont(QStringLiteral("Segoe UI")).exactMatch() ? QStringLiteral("Segoe UI") : QStringLiteral("Roboto");
#else
    return QStringLiteral("Roboto");
#endif
}

class Fonts {
public:
    static Fonts &instance()
    {
        static Fonts fonts;
        return fonts;
    }

    const QFont &titleFont() const { return m_titleFont; }

private:
    Fonts() :
#ifdef __APPLE__
        m_titleFont(displayFont(), 13, 65)
#else
        m_titleFont(displayFont(), 10, 60)
#endif
    {}

    QFont m_titleFont;
};
```
And then you can access the font like  `m_label->setFont(Fonts::instance().titleFont())`, for example.

(however, I would encourage you to open separate PRs for such a change, instead of trying to fit all the "Qt 6" related changes in this one)